pkg/templates/templates/create/porter.yaml

```diff
@@ -8,9 +8,6 @@ version: 0.1.0
 description: "An example Porter configuration"
 tag: getporter/porter-hello:v0.1.0
```

Comment:
Having a TODO to change the tag registry would be great, but as a follow-up PR is just fine. This is a chonking PR 😀

Reply:
😅 added!